c_test/c_test.vcxproj

```diff
@@ -217,4 +220,5 @@
     </PropertyGroup>
     <Error Condition="!Exists('$(SolutionDir)\.nuget\NuGet.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\.nuget\NuGet.targets'))" />
   </Target>
+  <Import Project="..\sdl\SDL-7.0-NativeAnalysis.targets" />

```

Comment:
Do we want to do this for all windows builds?

Reply:
Yes.  It degrades gracefully.  It will be ignored if SDL tools are not present.